packages/select/src/components/ui/ChipMultiSelect.tsx

```diff
@@ -66,7 +66,7 @@ export const ChipMultiSelect = React.memo<ChipMultiSelectProps>(
           {...selectProps}
           isClearable={false}
           value={value}
-          onChange={onValueChange}
+          onChange={onValueChange as MultiSelectProps["onChange"]}
```

Comment:
What if we use `ValueAndOnValueChangeProps<ReadonlyArray<ChipMultiSelectValue>> {` instead?

Reply:
Would require every one that uses it to use Readonly, for no good reason.